src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -3093,11 +3086,8 @@ public String toString() {
 
         @Override
         public int hashCode() {
-            int hashCode = Float.hashCode(value) | Short.hashCode(type);
-            if (units != null) {
-                hashCode |= units.hashCode();
-            }
-            return hashCode;
+            return Float.hashCode(value) | Short.hashCode(type)
+                   | Objects.hashCode(units);
```

Comment:
To be consistent with other similar methods above:
```suggestion
            return Float.hashCode(value)
                   | Short.hashCode(type)
                   | Objects.hashCode(units);
```